pkg/controllers/deprovisioning/helpers.go

```diff
@@ -209,6 +209,10 @@ func buildNodePoolMap(ctx context.Context, kubeClient client.Client, cloudProvid
 		if err != nil {
 			return nil, nil, fmt.Errorf("listing instance types for %s, %w", np.Name, err)
 		}
+		if len(nodePoolInstanceTypes) == 0 {
+			logging.FromContext(ctx).With("provisioner", np.Name).Debug("skipping provisioner, no resolved instance types found")
```

Comment:
```suggestion
			logging.FromContext(ctx).With(lo.Ternary(np.IsProvisioner, "provisioner", "nodepool"), np.Name).Debug("skipping, no resolved instance types found")
```

We need to handle the logging for both cases when we support NodePool